### ISSUE TITLE
Prediction Probability code explanation

### ISSUE THREAD

**Issue by Hardik93** [abhijeet3922/Face_ID#3]
Can you provide us with the explanation of your code's logic to calculate the probability while  comparing the input image with the align images. I want to modify that code so can you please provide guidance regarding the same.

Much thanks for the code.

**Comment by abhijeet3922** [comment]
Hope you have gone through this post where I have tried explaining each and every step. Still if there is anything which you want an explanation let me know.

https://appliedmachinelearning.blog/2018/10/30/yet-another-face-recognition-demonstration-on-images-videos-using-python-and-tensorflow/

**Comment by Hardik93** [comment]
Yes, I have gone through this blog and the code works fine. I just want to know where is the Euclidian Distance calculation/ implementation in code?

**Comment by rohitdoddapani** [comment]
> Yes, I have gone through this blog and the code works fine. I just want to know where is the Euclidian Distance calculation/ implementation in code?

Hi, bro needs some clarifications on running the code. I got stuck with some errors while training the model.

Got this
ValueError: Node 'gradients/InceptionResnetV1/Bottleneck/BatchNorm/cond/FusedBatchNorm_1_grad/FusedBatchNormGrad' has an _output_shapes attribute inconsistent with the GraphDef for output #3: Dimension 0 in both shapes must be equal, but are 0 and 512. Shapes are [0] and [512].
